README.md

```diff
@@ -1,63 +1,3280 @@
+    Dec 03 11:46:13 11a12ec openvpn[1448]: Tue Dec  3 11:46:13 2019 /etc/openvpn-misc/upscript.sh resin-vpn 1500 1555 10.240.53.172 52.4.252.97 init
+    Dec 03 11:46:13 11a12ec openvpn[1448]: resin-ntp-config: Found config.json in /mnt/boot/config.json .
+    Dec 03 11:46:13 11a12ec openvpn[1448]: Tue Dec  3 11:46:13 2019 /sbin/ip route add 52.4.252.97/32 via 52.4.252.97
+    Dec 03 11:46:13 11a12ec openvpn[1448]: ip: RTNETLINK answers: File exists
+    Dec 03 11:46:13 11a12ec openvpn[1448]: Tue Dec  3 11:46:13 2019 ERROR: Linux route add command failed: external program exited with error status: 2
+    Dec 03 11:46:13 11a12ec openvpn[1448]: Tue Dec  3 11:46:13 2019 GID set to openvpn
+    Dec 03 11:46:13 11a12ec openvpn[1448]: Tue Dec  3 11:46:13 2019 UID set to openvpn
+    Dec 03 11:46:13 11a12ec openvpn[1448]: Tue Dec  3 11:46:13 2019 Initialization Sequence Completed
+    ```
+* `systemctl start <serviceName>` - Will start a non-running service. Note that
+    this will *not* restart a service that is already running.
+* `systemctl stop <serviceName>` - Will stop a running service. If the service
+    is not running, this command will not do anything.
+* `systemctl restart <serviceName>` - Will restart a running service. If the
+    service is not running, this will start it.
+* `systemctl daemon-reload` - Will essentially run through the startup process
+    systemd carries out at initialisation, but without restarting services or
+    units. This allows the rebuild of the system dependency graph, and should be
+    carried out if any of the unit files are changed whilst the system is
+    running.
+
+This last command may sound a bit confusing, but take a quick example. Imagine
+that you need to dynamically change the `resin-supervisor.service` unit file
+for some reason (maybe to increase the healthcheck timeout on a very slow
+system). Once that change has made, you'll want to restart the service. However,
+first, you need to reload the unit file as this is changed from the loaded
+version. To do this, you'll run `systemctl daemon-reload` and then
+`systemctl restart resin-supervisor.service` to restart the Supervisor.
+
+In general, there are some core services that need to execute for a device to
+come online, connect to the balenaCloud VPN, download applications and then run
+them:
+
+* `chronyd.service` - Responsible for NTP duties and syncing 'real' network
+    time to the device.
+* `dnsmasq.service` - The local DNS service which is used for all host OS
+    lookups (and is the repeater for application service containers by default).
+* `NetworkManager.service` - The underlying Network Manager service, ensuring
+    that configured connections are used for networking.
+* `os-config.service` - Retrieves settings and configs from the API endpoint,
+    including certificates, authorised keys, the VPN config, etc.
+* `openvpn.service` - The VPN service itself, which connects to the balenaCloud
```

Comment:
consider including the old name as well. I think it used to be called `openvpn-resin.service`. Not super sure when it changed.

Reply:
Done (v2.10.0).